Makefile

```diff
@@ -27,21 +27,24 @@ endif
 .PHONY: test-unit
 
 images:
-	imagebuilder -f Dockerfile -t openshift/origin-cluster-kube-apiserver-operator .
+	imagebuilder -f Dockerfile -t $(DOCKER_ORG)/origin-cluster-kube-apiserver-operator .
 .PHONY: images
 
 clean:
 	$(RM) ./cluster-kube-apiserver-operator
 .PHONY: clean
 
+, := ,
+IMAGES ?= cluster-kube-apiserver-operator
+QUOTED_IMAGES=\"$(subst $(,),\"$(,)\",$(IMAGES))\"
+
 origin-release:
 	docker pull registry.svc.ci.openshift.org/openshift/origin-release:v4.0
-	bash -c 'docker build -f <(sed "s/DOCKER_ORG/$(DOCKER_ORG)/" Dockerfile-origin-release) -t "$(DOCKER_ORG)/origin-release:latest" .'
+	bash -c 'docker build -f <(sed "s/DOCKER_ORG/$(DOCKER_ORG)/g;s/IMAGES/$(QUOTED_IMAGES)/g" hack/lib/Dockerfile-origin-release) -t "$(DOCKER_ORG)/origin-release:latest" hack'
 	docker push $(DOCKER_ORG)/origin-release:latest
 	@echo
 	@echo "To install:"
 	@echo
-	@echo "  make images"
-	@echo "  docker tag openshift/origin-cluster-kube-apiserver-operator $(DOCKER_ORG)/origin-cluster-kube-apiserver-operator"
+	@echo "  DOCKER_ORG=$(DOCKER_ORG) make images"
```

Comment:
don't you need this before running this so you have the correct SHA?

Reply:
no, we use latest, no SHA